src/rgw/rgw_keystone.cc

```diff
@@ -16,7 +16,9 @@
 
 #define dout_subsys ceph_subsys_rgw
 
-int rgw_open_cms_envelope(CephContext * const cct, string& src, string& dst)
+int rgw_open_cms_envelope(CephContext * const cct,
```

Comment:
The line, after applying the `const` modifier, would fairly excess the 80-chars limit.